File: mobile/android/core/src/main/kotlin/com/portalia/core/config/AppConfig.kt
package com.portalia.core.config

/**
 * Aponta para o backend Portalia. `10.0.2.2` é o alias que o emulador Android
 * usa para "o localhost da máquina que roda o emulador" — troque para o IP
 * real na rede ao testar num aparelho físico.
 */
object AppConfig {
    const val API_BASE_URL = "http://10.0.2.2:3101/api/v1/"
}

File: mobile/ios/PortaliaCore/Sources/PortaliaCore/Config/AppConfig.swift
import Foundation

/// Aponta para o backend Portalia. Troque `baseURL` por build configuration
/// (dev/staging/prod) quando houver múltiplos ambientes reais — hoje só existe
/// o dev local, então fica hardcoded num único lugar.
public enum AppConfig {
    public static let apiBaseURL = URL(string: "http://localhost:3101/api/v1")!
}
